cvat/apps/engine/media.mimetypes

```diff
@@ -107,6 +107,7 @@ image/jp2	jp2
 image/x-portable-bitmap	pbm
 image/x-eps	epsf
 image/x-photo-cd	pcd
+image/bin	bin
```

Comment:
Probably we need two media types in accordance with https://en.wikipedia.org/wiki/Media_type
image/x.kitti-velodyne bin
image/x.point-cloud-data pcd

Reply:
We have added the two media types as suggested, but in the code there is an existing mime-type for pcd i.e 'image/x-photo-cd	pcd', kindly let us know if the current one will still be applicable.

Reply:
Let's get rid of old one

Reply:
Thanks, Made changes accordingly.